recipe/conda_build_config.yaml

```diff
@@ -1,5 +1,5 @@
 mpi_type:
-  - external
+  - external  # [linux and x86_64]
```

Comment:
One more question: Why do we only limit to linux64?

Reply:
(still trying to catch up the recent development in this feedstock...)

Reply:
just so it builds only once, since the package is the same on all platforms.

Reply:
ah ok, I thought by marking `noarch` conda-build or conda-smithy would figure this out, but sounds like we have to hack it ourselves 😅

Reply:
Ah, yes. It would, were it not for the other outputs which _aren't_ noarch. If it were just a single (or all) noarch package, it would figure it out. It's really _how_ we are adding the noarch version via a build matrix that means we have to avoid expanding the cartesian product of `mpi_type` and platform, which is conda-build's standard behavior for multiple values.